packages/api/src/graphql/typeDefs/index.js

```diff
@@ -115,6 +115,18 @@ const typeDefs = gql`
     loo: Loo
   }
 
+  """
+  A container type for various statistical counters
+  """
+  type Counters {
```

Comment:
It'b be totally amazing to document these properties... If you can figure from the logic what they _really_ mean ;-)